WordPress/src/main/java/org/wordpress/android/push/GCMMessageHandler.java

```diff
@@ -473,7 +473,8 @@ private void addActionsForCommentNotification(Context context, NotificationCompa
                 // if the comment is lacking approval, offer moderation actions
                 if (note.getCommentStatus() == CommentStatus.UNAPPROVED) {
                     if (note.canModerate()) {
-                        addCommentApproveActionForCommentNotification(context, builder, noteId);
+                        // TODO enable comment approve action after fixing content lost after approval
```

Comment:
I'm wondering if we could reference in the comment the GitHub ticket related to this ([reference](https://github.com/wordpress-mobile/WordPress-Android/issues/17026)) for adding extra context, wdyt?

Reply:
Good idea, thanks

Added issue link